app/models/papi_client.rb

```diff
@@ -219,6 +227,11 @@ def set_environment_variables
       'SENTRY_DSN' => ENV['SENTRY_DSN'],
       'BARD_HOST_URL' => Rails.application.config.bard_host_url
     }
+    if action == :image_pipeline
+      vars.merge({ 'IS_PAPI' => '1', 'NODE_TLS_REJECT_UNAUTHORIZED' => '0' })
```

Comment:
Per other comment on TLS and PAPI:

```suggestion
      vars.merge({ 'IS_PAPI' => '1'})
```

Reply:
Ah, yes.  A higher-fidelity manual test endpoint might be "Confirm the study's Google bucket contains .webp image files in its `_scp_internal/cache/expression_scatter/images` directory."